src/main/resources/application.yaml

```
@@ -0,0 +1,10 @@
+spring:
+  application:
+    name: demo
+  jpa:
+    database: POSTGRESQL
+    hibernate:
+      ddl-auto: update
```

```suggestion
      ddl-auto: validate
```
Tror vi sa det att vi skulle använda liquibase XML för schema-skapande